catboost/libs/algo/tensor_search_helpers.cpp

```diff
@@ -363,7 +364,8 @@ void Bootstrap(
     const float baggingTemperature = params.ObliviousTreeOptions->BootstrapConfig->GetBaggingTemperature();
     const float takenFraction = params.ObliviousTreeOptions->BootstrapConfig->GetTakenFraction();
     const bool isPairwiseScoring = IsPairwiseScoring(params.LossFunctionDescription->GetLossFunction());
-    const float headFraction = params.ObliviousTreeOptions->BootstrapConfig->GetMvsHeadFraction();
+    const float mvsReg = params.ObliviousTreeOptions->BootstrapConfig->GetMvsReg();
+    const bool mvsRegIsSet = params.ObliviousTreeOptions->BootstrapConfig->MvsRegIsSet();
     bool isCoinFlipping = true;
```

Comment:
please rename isCoinFlipping to useRandomSampling or something else, which would be meaningful